### PR TITLE
Window scroll

### PR DESCRIPTION
**Pull request by ganqqwerty**
**Summary**

This PR fixes the scrolling in the window container. With it, the scrollDistance in window element works the same way as scrollDistance in div: 0 for loading more results when the slider is at very bottom, 1 - when it's 10% to the bottom, etc.

### CODE REVIEW

**Comment by riz96** [comment]
Hi! Have you got any update about this PR? Thanks :)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-infinite-scroll",
-  "version": "7.1.0",
+  "version": "8.0.0",
```
Comment (orizens) [inline package.json]: please remove version update from here.
it's being updated when on release.

File: src/services/position-resolver.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ export function calculatePointsForWindow(
   const totalToScroll =
     getElementOffsetTop(element.nativeElement, axis, isWindow) +
     nativeElementHeight;
-  return { height, scrolled, totalToScroll };
+  return { height, scrolled, totalToScroll, isWindow: true };
```
Comment (orizens) [inline src/services/position-resolver.ts]: "isWindow" is already defined. can you reuse it?

Comment (ganqqwerty) [reply src/services/position-resolver.ts]: oh, sure

File: package.json (hunk 1 above)
Comment (ganqqwerty) [reply package.json]: okay, will do!

**Comment by riz96** [comment]
Hello! Still no updates? Thx.

**Comment by ganqqwerty** [comment]
@orizens please check it again. I updated the pull request

**Comment by orizens** [comment]
thanks @ganqqwerty.
that's released with 7.2.0

**Pull request merged**